frigate/video.py

```diff
@@ -236,6 +236,10 @@ def add_objects(self, objects):
         for obj in objects:
             if obj['name'] == 'person':
                 person_area = (obj['xmax']-obj['xmin'])*(obj['ymax']-obj['ymin'])
+
+                # Store person_area to use in the label later
+                obj['area'] = person_area
```

Comment:
I think it would be best to move the area calculation up above the `if` statement and calculate area for all objects, not just persons. Then we can use that one value everywhere.

Reply:
Good catch. I thought `self.detected_objects.append(obj)` was in the same scope of the if statement. Resolved!